tests/test_phygnn_model.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
How do you feel about adding a save model test? Take a look at the save load tests in test_phygnn.py. Might be good since a big part of this is to make sure we can efficiently save and load the norm params and start predicting right away.

Reply:
can do. I thought we had a saved model test somewhere, maybe in Gooml